demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -38,20 +43,19 @@ def are_release_notes_complete(self):
                     file_name, file_type = update_rn_util.identify_changed_file_type(file)
                     if file_name and file_type:
                         if (file_type not in self.latest_release_notes) and (file_name not in self.latest_release_notes):
-                            print_error(f"No release note entry was found for a {file_type.lower()} in the {self.pack_name} pack. "
-                                        f"Please rerun the update-release-notes command without -u to generate an"
-                                        f" updated template.")
-                            is_valid = False
+                            error_message, error_code = Errors.missing_release_notes_entry(file_type, self.pack_name)
+                            if self.handle_error(error_message, error_code, self.file_path):
```

Comment:
Maybe this way is better?
```suggestion
                            if self.handle_error(error_message, error_code, self.pack_name):
```

Reply:
this is ok it points to the file and indicated the change - looks like so: 
![image](https://user-images.githubusercontent.com/50295826/82751915-3b761780-9dc3-11ea-8d9b-0a25ab8939a4.png)